lib/datadog/tracing/contrib/dalli/instrumentation.rb

```diff
@@ -54,6 +54,9 @@ def request(op, *args)
                   cmd = Quantize.format_command(op, args)
                   span.set_tag(Ext::TAG_COMMAND, cmd)
                 end
+                # BEGIN BRAZE MODIFICATION
+                span.set_tag(Ext::TAG_LENGTH, Core::Utils.utf8_encode([op, *args].join(' ').strip, binary: true).length)
```

Comment:
the `Quantize.format_command(op, args)` above
already ran `Core::Utils.utf8_encode(command, binary: true, placeholder: placeholder)`

I think it would be better to only run that once, the cost of this observability is pretty high already for the use of dalli, which gets used all the time.

Also, should this tag only be added when `datadog_configuration[:command_enabled]` is true?